RELEASING.md

```diff
@@ -2,28 +2,11 @@ Release Guide
 =============
 
 1. Ensure that pulpcore of the corresponding version is available on PyPI.
-1. Generate changelog (`towncrier --yes --version 3.8.0`).
-1. Update version in the `galaxy.yml` and in the `roles/pulp_common/{vars,defaults}/main.yml`.
-1. Update references to current and previous versions in `docs/index.md`
-   (`sed -i -e 's/3.7.1/3.8.0/g' -e 's/3.6.z/3.7.z/g' -e 's/3.5.z/3.6.z/g' docs/index.md`).\
```

Comment:
Those changes are not handled by bump2version, are they?
Maybe we should keep that line.

Reply:
instead of using `bump2version` I added some `input()` asking for the new versions, I get the old ones and do a `sed`, so the idea is just to check if the `sed` did change the right places before starting the PR